### PR TITLE
US127227 - CPD - Edit Targets Dialog Should Default to Current Values

### PR DESCRIPTION
**Pull request by Syeda-Azim**
https://rally1.rallydev.com/#/431372673576d/iterationstatus?detail=%2Fuserstory%2F521907501836

### CODE REVIEW

File: src/components/cpd-manage-targets.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ class ManageCpdTargets extends BaseMixin(LitElement) {
 						class="numberInput"
 						required type="number"
 						min="0"
-						value="${this.dialogData.StructuredMinutes}"
+						value="${this.dialogData.StructuredRecordCount}"
```
Comment (imcdonald-d2l) [inline src/components/cpd-manage-targets.js:240]: Don't we want a condition here to grab either `StructuredMinutes` or `StructuredRecordCount` based on the config set?

File: data/job_title_target.json [hunk 2]
```diff
@@ -1,65 +1,78 @@
 {
-  "TargetId":1,
-  "Type":"JobTitle",
-  "JobTitle":"Sous-Chef",
-  "UserId":175,
-  "StartDate":"0001-12-17T00:00:00.000Z",
-  "Subjects":[
+  "TargetId": 1,
```
Comment (imcdonald-d2l) [inline data/job_title_target.json:2]: I don't have context on where `job_title_target.json` and `personal_target.json` are used. Is it just fake data used for testing components?

Comment (Syeda-Azim) [reply data/job_title_target.json:2]: These json files contains fake data for demo purpose.

File: src/components/cpd-manage-targets.js (hunk 1 above)
Comment (Syeda-Azim) [reply src/components/cpd-manage-targets.js:240]: The condition is at the beginning of this method and there is a seperate hmtl to render Record Duration.

**Review by imcdonald-d2l — APPROVED** [review]

**Pull request merged**